### DISCUSSION TITLE
[Feature] Proper behavior for page-up and page-down keys (web)

### DISCUSSION THREAD

**Discussion by erikvanoosten** [immich-app/immich#9909] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

On Firefox 12.0.1 (mac), the page-up / page-down keys bring you to the top/bottom of the timeline. If would be great if those keys would make the timeline scroll only 1 page up or down.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#6202

**Comment by erikvanoosten** [comment]
Thanks for finding the duplicate. Apparently I used the wrong search keywords.
